docker-compose.yml

```diff
@@ -58,8 +58,10 @@ services:
       interval: 1m
       timeout: 10s
       retries: 5
-      start_period: 10s
```

Comment:
It looks like we should be able to use this, is it useful on its own: https://docs.docker.com/compose/compose-file/compose-versioning/#version-34?

Reply:
Nope, because the interval is set to 1 minute, so no health-check will be applied during that period.